src/Microsoft.SqlTools.ManagedBatchParser/BatchParser/BatchParserWrapper.cs

```diff
@@ -229,7 +229,7 @@ private static List<int> GetOffsets(string content, IList<BatchInfo> batchInfos)
             }
 
             // get number of characters in the last line
-            int endColumn = prevLine.ToCharArray().Length;
+            int endColumn = prevLine == null ? 0 : prevLine.ToCharArray().Length;
```

Comment:
Update comment explaining what the situation would be where this would be null

Reply:
updated comment